integration-tests/integration/cellMove/viewCellSharingRiskAssessmentDetails.spec.js

```diff
@@ -0,0 +1,77 @@
+const offenderFullDetails = require('../../mockApis/responses/offenderFullDetails')
+const CellSharingRiskAssessmentPage = require('../../pages/cellMove/cellSharingRiskAssessmentDetailsPage')
+
+const offenderNo = 'A12345'
+
+context('A user can view non associations', () => {
+  before(() => {
+    cy.clearCookies()
+    cy.task('reset')
+    cy.task('stubLogin', { username: 'ITAG_USER', caseload: 'MDI' })
+    cy.login()
+  })
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('hmpps-session-dev')
+    cy.task('resetAndStubTokenVerification')
+    cy.task('stubOffenderFullDetails', offenderFullDetails)
+    cy.task('stubCsraAssessments', {
+      offenderNumbers: [offenderNo],
+      code: 'CSR',
+      assessments: [
+        {
+          bookingId: 1234,
+          offenderNo,
+          classificationCode: 'STANDARD',
+          classification: 'Standard',
+          assessmentCode: 'CSR',
+          assessmentDescription: 'CSR Rating',
+          cellSharingAlertFlag: true,
+          assessmentDate: '2020-08-18',
+          nextReviewDate: '2020-08-19',
+          approvalDate: '2020-08-18',
+          assessmentAgencyId: 'MDI',
+          assessmentStatus: 'A',
+          assessmentSeq: 1,
+          assessmentComment: 'Some comment',
+          assessorId: 1,
+          assessorUser: 'TEST_USER',
+        },
+      ],
+    })
+  })
+
+  it('Shows the correct data for non-associations', () => {
+    cy.visit(`/prisoner/${offenderNo}/cell-move/cell-sharing-risk-assessment-details`)
+    const cellSharingRiskAssessmentDetailsPage = CellSharingRiskAssessmentPage.verifyOnPage()
+    cy.get('.govuk-summary-list--no-border').then($section => {
```

Comment:
Should this be encapsulated inside the page object?

Reply:
we haven't been doing that for summaries as various parts of them are optional. Less so in this one, so probably wouldn't hurt. I'll update when next in the area, I'm sure it won't be long